CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Unreleased
 - Make OAuthLogin component responsive [#28](https://github.com/CartoDB/carto-react-lib/pull/28)
 
+## 1.0.0-beta5 (2020-11-26)
+- Added meta value version to List component [#31](https://github.com/CartoDB/carto-react-lib/pull/31)
```

Comment:
Move this item to ##Unreleased section, please

Reply:
done